satellite/peer.go

```diff
@@ -16,6 +18,10 @@ import (
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/grpc"
 
+	m "storj.io/storj/internal/mail"
+	"storj.io/storj/internal/mail/oauth2"
+	m1 "storj.io/storj/satellite/mail"
```

Comment:
Don't use single letter aliases. They span the whole file and understanding `m1` in arbitrary location is difficult to understand.